core-bundle/src/Resources/config/listener.yml

```diff
@@ -88,6 +88,28 @@ services:
         tags:
             - { name: contao.hook, hook: loadDataContainer }
 
+    contao.listener.data_container.element_template_options:
+        class: Contao\CoreBundle\EventListener\DataContainer\TemplateOptionsListener
+        arguments:
+            - '@contao.framework'
+            - '@request_stack'
+            - '%contao.content_element.templates%'
+            - ce_
+            - Contao\ContentProxy
+        tags:
+            - { name: contao.callback, table: tl_content, target: fields.customTpl.options }
```

Comment:
Why are you not using the `@Callback` annotation here?

Reply:
Hm, I don't think that works for reusable services like this?

Reply:
You are right. Sorry. 🙃